### PR TITLE
Move specific webpack version to devDependencies

### PR DESCRIPTION
**Pull request by sirlancelot**
I have `webpack@3` in my project. After adding `webpack-dashboard@1.0.0-5`, I now have two versions of webpack in my `node_modules` tree. Since `webpack` is already listed as a peer dependency, a specific version should only be added as a devDependency.

Thanks! I'm loving working at NASA with this dashboard! :tada:

### CODE REVIEW

**Comment by ryan-roemer** [comment]
LGTM. 

@kenwheeler @tptee ?

**Comment by sirlancelot** [comment]
I updated `package-lock.json` as well as placed `webpack-dev-server` as a peerDependency.

File: package.json [hunk 1]
```diff
@@ -25,7 +25,8 @@
   },
   "homepage": "https://github.com/FormidableLabs/webpack-dashboard#readme",
   "peerDependencies": {
-    "webpack": "*"
+    "webpack": "*",
+    "webpack-dev-server": "*"
```
Comment (ryan-roemer) [inline package.json]: Is this supposed to be a `peerDependency`? We don't `require` `webpack-dev-server` anywhere in the code...

Comment (sirlancelot) [inline package.json]: My bad. If it's used at all then perhaps I should remove it from devDependencies as well?

**Comment by sirlancelot** [comment]
Removed `webpack-dev-server` peerDependency. It is still a devDependency.

**Comment by ryan-roemer** [comment]
Heyo Dr. @kenwheeler -- Is `webpack-dev-server` actually needed/used at all as a `devDependency`? I don't see any code / cli uses, just a suggestion snippet in a doc for upstream users...

**Comment by sirlancelot** [comment]
`webpack-dev-server` isn't needed, should I remove it in order to get this PR merged? Thanks!

**Comment by tptee** [comment]
Looks like it's safe to remove `webpack-dev-server` entirely, would be much appreciated!

**Comment by ryan-roemer** [comment]
Thanks @tptee !

@sirlancelot -- Yep, remove `webpack-dev-server`, regen yarn.lock and I think we're good to go! Thanks!

**Comment by sirlancelot** [comment]
I removed `webpack-dev-server` from the list of dependencies and rebuilt both yarn and npm lockfiles.

**Comment by ryan-roemer** [comment]
@tptee -- If you get a free moment can you do a release?

**Comment by sirlancelot** [comment]
Thanks! :tada:

**Pull request merged**